README.md

```diff
@@ -232,35 +271,30 @@ As Device Farm supports additional test types the plugin must be extended to ena
   1. If the test type requires a test artifact package to be uploaded in addition to the app then make sure the new class extends the TestPackageProvider trait.
   2. If the test type supports the 'filter' parameter extend the HasFilter trait.
 3. Modify DeviceFarmExtension to add a method for the new test type, like so:
-```
-    void mynewtesttype(final Closure closure) {
-        NewTestType newTest = new NewTestType()
-        project.configure newTest, closure
-        test = newTest
-    }        
+```groovy
+void mynewtesttype(final Closure closure) {
+    NewTestType newTest = new NewTestType()
+    project.configure newTest, closure
+    test = newTest
+}        
 ```
 * This will enable the configuration of the new test type within the devicefarm plugin.
 
+Build Dependencies
+------------------
 
-Dependencies
-============
+### Java SDK (Android Studio Project)
 
-
-Java SDK (Android Studio Project)
---------
 * Java 7 or higher
 
+### Runtime (Include these in your `build.gradle`)
 
-Runtime (Include these in your build.gradle)
---------------------------------------------
-
-* AWS SDK 1.10.15 or later.
+* AWS SDK 1.11.240 or later
 * Android tools builder test api 0.5.2
 * Apache Commons Lang3 3.3.4
 
-For Unit tests (Include these in your build.gradle)
----------------------------------------------------
- 
+### Unit Tests (Include these in your `build.gradle`)
```

Comment:
We should mention which build.gradle file

Reply:
These dependencies are for building the plugin, as opposed to the app being tested. 

To be honest, I don't think it adds any real value by defining the build dependencies here. They're already defined in the plugin's `build.gradle` file. 

It seems like the only thing this section does is confuse users. You okay with me removing the Build Dependencies section?